projects/packages/connection/legacy/class-jetpack-xmlrpc-server.php

```diff
@@ -823,24 +823,6 @@ public function disconnect_blog() {
 		return false;
 	}
 
-	/**
```

Comment:
I removed this long deprecated method which was calling the now deprecated method `Jetpack_XMLRPC_Methods::features_available`.